UnityProject/Assets/Scripts/Systems/Occupations/OccupationCustomEffects/OccupationCustomEffectBase.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[System.Serializable]
+public abstract class OccupationCustomEffectBase
```

Comment:
looks like this might as well be an interface. Do you have plans on adding members to this?

Reply:
I think it won't work if it's an interface, with the editor inspector